scripts/create-hsm-wallet.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { hsm } = require('@dinamonetworks/hsm-dinamo');
+const { isNumber } = require('lodash');
+const settings = require('../src/settings');
+
+/**
+ * Creates a new xPriv key on the HSM
+ * @param {string} keyName Key name to be created on the HSM
+ */
+async function createXprivKey(keyName) {
+  const config = settings.getConfig();
+  // Gets the connection data from the global config file
+  const hsmConnectionOptions = {
+    host: config.hsmHost,
+    authUsernamePassword: {
+      username: config.hsmUsername,
+      password: config.hsmPassword,
+    },
+  };
+  const hsmConnection = await hsm.connect(hsmConnectionOptions);
+
+  // Creates a key
+  const key = await hsmConnection.blockchain.create(
+    keyName, // Name of the key
+    hsm.enums.BLOCKCHAIN_KEYS.BIP32_XPRV,
+    false, // Exportable
+    false, // Temporary
+    hsm.enums.VERSION_OPTIONS.BIP32_MAIN_NET
```

Comment:
Is this the network version byte parameter?

Reply:
Yes, added some jsdocs to clarify this on [15bb806](https://github.com/HathorNetwork/hathor-wallet-headless/pull/357/commits/15bb80644137450b95b718f4250567ee3633d2e2)